client/src/app/components/nova/address/AccountAddressView.tsx

```diff
@@ -1,17 +1,26 @@
 import { AccountAddress } from "@iota/sdk-wasm-nova/web";
-import React from "react";
+import React, { Reducer, useReducer } from "react";
 import { useAccountAddressState } from "~/helpers/nova/hooks/useAccountAddressState";
 import Spinner from "../../Spinner";
 import Bech32Address from "../../nova/address/Bech32Address";
-import AssociatedOutputs from "./section/association/AssociatedOutputs";
+import { AddressPageTabbedSections } from "./section/AddressPageTabbedSections";
 
 interface AccountAddressViewProps {
     accountAddress: AccountAddress;
 }
 
+export interface IAccountAddressViewState {
+    isAssociatedOutputsLoading: boolean;
+}
+
 const AccountAddressView: React.FC<AccountAddressViewProps> = ({ accountAddress }) => {
     const { accountAddressDetails, isAccountDetailsLoading } = useAccountAddressState(accountAddress);
-    const isPageLoading = isAccountDetailsLoading;
+    const [state, setState] = useReducer<Reducer<IAccountAddressViewState, Partial<IAccountAddressViewState>>>(
```

Comment:
`useReducer` seams like an overkill here. Should we just use a `useState` ?

Reply:
Ah, i think i see now. I think we should push this `isAssociatedOutputsLoading` into each `useAddressTypeAddressState` and expose `[state, setState]` like we have on stardust.